libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -212,35 +242,41 @@ bool AC_PolyFence_loader::breached(const Vector2f& pos_cm) const
     // check we are inside each inclusion zone:
     for (uint8_t i=0; i<_num_loaded_inclusion_boundaries; i++) {
         const InclusionBoundary &boundary = _loaded_inclusion_boundary[i];
-        if (Polygon_outside(pos_cm, boundary.points, boundary.count)) {
+        if (Polygon_outside(pos, boundary.lla_points, boundary.count)) {
             return true;
         }
     }
 
     // check we are outside each exclusion zone:
     for (uint8_t i=0; i<_num_loaded_exclusion_boundaries; i++) {
         const ExclusionBoundary &boundary = _loaded_exclusion_boundary[i];
-        if (!Polygon_outside(pos_cm, boundary.points, boundary.count)) {
+        if (!Polygon_outside(pos, boundary.lla_points, boundary.count)) {
             return true;
         }
     }
 
-    // check circular excludes
+    Location loc;
+    loc.lat = pos.x;
+    loc.lng = pos.y;
+
     for (uint8_t i=0; i<_num_loaded_circle_exclusion_boundaries; i++) {
         const ExclusionCircle &circle = _loaded_circle_exclusion_boundary[i];
-        const Vector2f diff_cm = pos_cm - circle.pos_cm;
-        const float diff_cm_squared = diff_cm.length_squared();
-        if (diff_cm_squared < sq(circle.radius*100.0f)) {
+        Location circle_center;
+        circle_center.lat = circle.loc.x;
+        circle_center.lng = circle.loc.y;
+        const float diff_cm = loc.get_distance(circle_center)*100.0f;
+        if (diff_cm < circle.radius * 100.0f) {
             return true;
         }
     }
 
-    // check circular includes
     for (uint8_t i=0; i<_num_loaded_circle_inclusion_boundaries; i++) {
         const InclusionCircle &circle = _loaded_circle_inclusion_boundary[i];
-        const Vector2f diff_cm = pos_cm - circle.pos_cm;
-        const float diff_cm_squared = diff_cm.length_squared();
-        if (diff_cm_squared > sq(circle.radius*100.0f)) {
+        Location circle_center;
+        circle_center.lat = circle.loc.x;
+        circle_center.lng = circle.loc.y;
+        const float diff_cm = loc.get_distance(circle_center)*100.0f;
+        if(diff_cm > circle.radius * 100.0f) {
```

Comment:
minor formatting issue, we usually have a space between the "if" and the bracket.

Reply:
Ack - updated